ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-cpou-upgrade-from-4.15.yaml

```diff
@@ -48,6 +48,12 @@ releases:
       architecture: amd64
       channel: fast
       version: "4.15"
+  source:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.16"
```

Comment:
```suggestion
      version: "4.15"
```

Reply:
same to https://github.com/openshift/release/pull/57024/files#r1772434496